client/packages/system/src/ContactTrace/DetailView/DetailView.tsx

```diff
@@ -140,15 +140,19 @@ export const DetailView: FC<DetailViewProps> = ({
         </span>
       );
     }
-  }, [contactData]);
+  }, [contactData, setSuffix]);
```

Comment:
From react-hooks/exhaustive-deps. There are more deps missing, but they have issues as well, i.e. they re-render too often leading to infinite loops. The linter revealed it nicely, but I haven't fixed it yet...